__tests__/cmds/openapi/reduce.test.ts

```diff
@@ -17,8 +17,6 @@ const getCommandOutput = () => consoleInfoSpy.mock.calls.join('\n\n');
 
 describe('rdme openapi:reduce', () => {
   beforeEach(() => {
-    jest.mock('fs');
```

Comment:
Yeah I was looking for `fs` mock examples for my tests and I noticed this. I don't think it's necessary since we're also doing this: https://github.com/readmeio/rdme/blob/3d081794d6bb17dff11aef274a46f6e727ec33ca/__tests__/cmds/openapi/reduce.test.ts#L42-L45

Tests appear to pass and the mocks appear to operate as expected 🤷🏽‍♂️